src/components/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+
+type Props = {
+  todos: Todo[];
+  filterBy: Filter;
+  handleFilterBy: (filter: Filter) => void;
+};
+
+export const Footer: React.FC<Props> = ({
```

Comment:
checklist №3 bad naming
```suggestion
export const Filters: React.FC<Props> = ({
```